components/learn/modules/ROOT/pages/deploy-and-interact.adoc

```diff
@@ -295,7 +295,7 @@ If you restarted ganache at any point, this script may fail. Restarting ganache
 If this happens, simply <<local-blockchain, start ganache>> and <<deploying-a-smart-contract, redeploy>> the `Box` contract.
 ====
 
-We can now actually interact with the contract by sending a transaction to it.
+We can also interact with the contract by sending a transaction to it.
```

Comment:
```suggestion
```

You suggestion made me realize this sentence didn't actually add anything :joy:

Reply:
Removed line